management/src/main/resources/reference.conf

```diff
@@ -44,9 +44,16 @@ akka.management {
     # `route-providers += "com.example.MyExtraHealthCheckRoutes"` in the library `reference.conf`.
     #
     # Should not be set by end user applications in 'application.conf', use the extensions property for that
-    #
-    route-providers = ${?akka.management.http.route-providers} []
+    route-providers =   ${?akka.management.http.route-providers} ["akka.management.http.HealthCheckRoutes"]
+
+    health-checks {
+
+      readiness-path = "ready"
+      liveness-path = "alive"
 
+      readiness-checks = ${?akka.management.http.health-checks.readiness} []
+      liveness-checks = ${?akka.management.http.health-checks.liveness}[]
```

Comment:
comparing this with the above `route-providers`, is it missing `-checks`? like this
```
      readiness-checks = ${?akka.management.http.health-checks.readiness-checks} []
      liveness-checks = ${?akka.management.http.health-checks.liveness-checks}[]
```

Reply:
Looks like I have that but maybe I didn't in the version you saw